packages/eslint-plugin-react-hooks/__tests__/ESLintRulesOfHooks-test.js

```diff
@@ -1031,6 +1203,14 @@ function classError(hook) {
   };
 }
 
+function useEventError(fn) {
+  return {
+    message:
+      `Functions created with React Hook "useEvent" must be invoked locally in the component it's ` +
```

Comment:
nit: invoked -> called

"that is being passed by reference" -- it's unclear that by this part you're saying it's wrong. maybe:

>Functions created with React Hook "useEvent" can only be called from the same component. They cannot be assigned to variables or passed down.